pyproject.toml

```diff
@@ -33,7 +33,7 @@ build-backend = "poetry.core.masonry.api"
 
 [tool.black]
 line-length = 128
-target-version = ['py39', 'py310', 'py311']
+target-version = ['py39', 'py310', 'py311', 'py312']
 extend-exclude = "^/uai_openlabel"
```

Comment:
I somehow think this line `extend-exclude = "^/uai_openlabel"` was wrongly copied over from the aveas_openlabel repo, where this was a submodule we wanted to exclude. 

Could you please remove this line and run black again?